spec/integration/result_spec.rb

```diff
@@ -50,6 +50,33 @@
     end
   end
 
+  describe "#error?" do
+    subject { result }
+
+    let(:schema_result) do
+      double(:schema_result, message_set: [], to_h: {email: "jane@doe.org"})
+    end
+
+    let(:result) do
+      Dry::Validation::Result.new(schema_result) do |r|
+        r.add_error(Dry::Validation::Message.new("root error", path: [nil]))
+        r.add_error(Dry::Validation::Message.new("email error", path: [:email]))
+      end
```

Comment:
this is basically copy & past from above. Not sure what feelings are around here for contexts and distance to `let`s :)